mapentity/static/mapentity/mapentity.helpers.js

```diff
@@ -104,3 +104,18 @@ function expandDatatableHeight() {
 function tr(s) {
     return MapEntity.i18n[s] || s;
 }
+
+
+function tinyMceInit(editor) {
+    // Overflow on characters count
+    editor.on('WordCountUpdate', function(event) {
+        if (("container" in event.target === true) && (maxCharacters > 0)) {
```

Comment:
`"container" in event.target` va retourner un booléen, donc `=== true` est superflu.
De même pour les sub parenthèses où ici il n'y a pas d'ambiguité avec les opérateurs.

Pour des raisons de lisibilité je mettrai la condition dans l'autre sens avec un early return ⬇️ 
```js
if (!"container" in event.target || !maxCharacters) {
 return;
}
var characters = event.wordCount.characters;
```
Mais c'est ok comme ça aussi 👍

Reply:
J'ai enlevé `=== true` :+1: 

Par contre je suis pas très à l'aise sur les early return en javascript, même si ça paraît plus élégant, donc je garde comme ça ;)